docs/tutorials/ad-hoc-simulation/planning-simulation.md

```diff
@@ -98,7 +98,19 @@ ros2 topic pub /autoware/engage autoware_auto_vehicle_msgs/msg/Engage "engage: t
 
     Changes to the `velocity` parameter will only affect objects placed after the parameter is changed.
 
-![set-dummy-car](images/planning/lane-following/place-dummy-car.png) 4. Delete any dummy objects placed in the view by clicking the `Delete All Objects` button in the toolbar.
+![set-dummy-car](images/planning/lane-following/place-dummy-car.png)
+
+4. Delete any dummy objects placed in the view by clicking the `Delete All Objects` button in the toolbar.
+
+5. Click the `Interactive` button in the toolbar to make the dummy object interactive.
+
+![set-interactive-dummy-car](images/planning/lane-following/check-interactive.png)
```

Comment:
In order to not break the numbering, this block also needs to be indented (I just learned about it thanks for https://stackoverflow.com/questions/18088955/markdown-continue-numbered-list).
```suggestion
    ![set-interactive-dummy-car](images/planning/lane-following/check-interactive.png)
```